Apps/APN/src/data/models/user.js

```diff
@@ -0,0 +1,7 @@
+export default class User {
+  constructor(email, { name, isGuest } = { name: '', isGuest: false }) {
```

Comment:
Can we get rid of the `isGuest` logic? The rest of the sample apps don't use that type of logic - all users consist of a name and email.

Reply:
[Removed it](https://github.com/customerio/customerio-reactnative/pull/166/commits/1f90eb5723fc4cfd3020343da6836c61527895e5)